backend/inspirehep/records/marshmallow/journals/base.py

```diff
@@ -24,4 +24,13 @@ class Meta:
 
 class JournalsPublicSchema(JournalsRawSchema):
     class Meta:
-        exclude = FIELDS_TO_EXCLUDE
+        exclude = FIELDS_TO_EXCLUDE + ["_private_notes"]
+
+
+class JournalsPublicListSchema(JournalsRawSchema):
+    public_notes = fields.Method("", dump_only=True) 
+    short_title = fields.Method("", dump_only=True) 
+    publisher = fields.Method("", dump_only=True)
+    journal_title = fields.Method("", dump_only=True)
+    urls = fields.Method("", dump_only=True)
+    papers_count = fields.Method("", dump_only=True)
```

Comment:
why you are using `fields.method` here if you're not using any method? Also, I would move this to `ListSchema` and leave this schema as it was (without `_private_notes` and `_collections`)